### PR TITLE
Add error handling

### PR DESCRIPTION
**Pull request by Barisimre**
Add if statements to see switches error so it is not just a sed =

### CODE REVIEW

File: roboteam_ai/src/treeinterp/BTFactory.cpp [hunk 1]
```diff
@@ -1,5 +1,3 @@
-#include <utility>
-
 #include <boost/filesystem.hpp>
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/treeinterp/BTFactory.cpp]: BOOST??? anyway,
put it in BTFactory.h

File: roboteam_ai/src/treeinterp/TreeInterpreter.cpp [hunk 2]
```diff
@@ -1,8 +1,11 @@
+#include <utility>
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/treeinterp/TreeInterpreter.cpp]: REEEEE utility ;-)

File: roboteam_ai/src/treeinterp/TreeInterpreter.cpp [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+
 //
 // Created by baris on 01/10/18.
 //
 
 #include "TreeInterpreter.h"
+#include "Switches.h"
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/treeinterp/TreeInterpreter.cpp]: .h

File: roboteam_ai/src/treeinterp/TreeInterpreter.cpp [hunk 4]
```diff
@@ -142,6 +157,15 @@ bt::Node::Ptr TreeInterpreter::tacticSwitch(std::string name, bt::Blackboard::Pt
     return node;
 }
 
+bool TreeInterpreter::lookInVector(std::vector<std::string> vector, std::string item, std::string type) {
+    auto tempVector = std::move(vector);
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/treeinterp/TreeInterpreter.cpp]: names in this function are kinda confusing

File: roboteam_ai/src/treeinterp/TreeInterpreter.cpp (hunk 3 above)
Comment (mrlukasbos) [reply roboteam_ai/src/treeinterp/TreeInterpreter.cpp]: Why include switches at all? RIP compile time....

File: roboteam_ai/src/treeinterp/JsonReader.cpp [hunk 5]
```diff
@@ -24,7 +24,7 @@ std::string JsonReader::getFilePath(std::string name) {
         }
         smallPath.append(word + "/");
     }
-    std::cout << "Smallpath: " << smallPath << std::endl;
+    //std::cout << "Smallpath: " << smallPath << std::endl;
```
Comment (mrlukasbos) [inline roboteam_ai/src/treeinterp/JsonReader.cpp]: remove this comment

**Pull request merged**